### PR TITLE
fix: send blocked content curation to feed service

### PR DESCRIPTION
**Pull request by rebelchris**
Pass through the blocked_content_curation options to feed service.

@pnvasanth We send an example as such (this is correct?):
```
{
  user_id: '0c3TpWWzGYHbOOFp9k3av',
  page_size: 9,
  offset: 0,
  cursor: undefined,
  allowed_post_types: [ 'article', 'share', 'freeform', 'video:youtube', 'collection' ],
  total_pages: 1,
  fresh_page_size: '3',
  blocked_sources: [ 'community', 'fd062672-63b7-4a10-87bd-96dcd10e9613' ],
  blocked_content_curations: [ 'listicle' ]
}
```

### CODE REVIEW

**Comment by pnvasanth** [comment]
> Pass through the blocked_content_curation options to feed service.
> 
> @pnvasanth We send an example as such (this is correct?):
> ```
> {
>   user_id: '0c3TpWWzGYHbOOFp9k3av',
>   page_size: 9,
>   offset: 0,
>   cursor: undefined,
>   allowed_post_types: [ 'article', 'share', 'freeform', 'video:youtube', 'collection' ],
>   total_pages: 1,
>   fresh_page_size: '3',
>   blocked_sources: [ 'community', 'fd062672-63b7-4a10-87bd-96dcd10e9613' ],
>   blocked_content_curations: [ 'listicle' ]
> }
> ```

Yeah correct 👍

**Review by omBratteng — APPROVED** [review]

File: src/common/feedGenerator.ts [hunk 1]
```diff
@@ -167,13 +167,20 @@ export const feedToFilters = async (
     membershipsByHide.hide.push(WATERCOOLER_ID);
   }
 
+  const blockedContentCuration = settings
+    .filter(
+      (setting) => setting.group === 'content_curation' && setting.options.type,
+    )
+    .map((setting) => setting.options.type);
```
Comment (capJavert) [inline src/common/feedGenerator.ts:174]: You can do it in one go with `reduce`. Just so feed is critical and we probably want to be optimized here?

**Review by capJavert — APPROVED** [review]

File: src/common/feedGenerator.ts (hunk 1 above)
Comment (rebelchris) [reply src/common/feedGenerator.ts:174]: Ah yeah good point I just lazy copy pasted another one above. 😅
Can refactor both tbh

**Comment by rebelchris** [comment]
@capJavert Added a reduce that actually does both in one go, let me know if that works.

**Pull request merged**